src/main/java/seedu/address/storage/patient/PatientRecordsStorage.java

```diff
@@ -21,7 +21,7 @@
     Path getPatientRecordsFilePath();
 
     /**
-     * Returns CovigentApp data as a {@code ReadOnlyList<Patient>}.
```

Comment:
i think this should be `{@code ReadOnlyList<Patient>}` instead of `{@code ReadOnlyList<Room>}`?